src/components/CustomButton.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+import { defineProps, defineEmits } from "vue";
+const props = defineProps({
+  iconSource: {
+    type: String,
+    default: "",
+  },
+  iconAlternativeText: {
+    type: String,
+    default: "",
+  },
+  label: {
+    type: String,
+    default: "",
+  },
+  disabled: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+defineEmits({
+  click: {},
+});
+
+function showIcon() {
+  return props.iconSource || props.iconAlternativeText;
```

Comment:
```suggestion
  return props.iconSource && props.iconAlternativeText;
```

Or you can be more liberal:
```suggestion
  return props.iconSource;
```

Reply:
You can also remove this function.